cli/commands/tool/tool_test.go

```diff
@@ -61,8 +62,9 @@ func TestMain(m *testing.M) {
 	}
 	orchestratorService = service_orchestrator.NewService()
 
-	sock, server, _, err = service.StartDedicatedAuthServer(":0")
-	orchestrator.RegisterOrchestratorServer(server, orchestratorService)
+	sock, server, _, err = service.StartDedicatedAuthServer(":0", func(server *grpc.Server) {
```

Comment:
We could add a `WithOrchestrator` to the service package to make this a little be cleaner